presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -0,0 +1,140 @@
+===============================
+Presto C++ Properties Reference
+===============================
+
+This section describes Presto C++ configuration properties.
+
+The following is not a complete list of all configuration and
+session properties, and does not include any connector-specific
+catalog configuration properties. For information on catalog 
+configuration properties, see :doc:`Connectors </connector/>`.
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Coordinator Properties
+----------------------
+
+Set the following properties in the Presto coordinator to enable its use of Presto C++ workers. 
+
+.. code-block:: none
+
+    native-execution-enabled=true
+    optimizer.optimize-hash-generation=false
+    regex-library=RE2J
+    use-alternative-function-signatures=true
+    experimental.table-writer-merge-operator-enabled=false
+
+``experimental.table-writer-merge-operator-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+  Merge TableWriter output before sending to TableFinishOperator.
+
+``native-execution-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+  This property is required when running Presto C++ workers because of 
+  underlying differences in behavior from Java workers.
+
+``optimizer.optimize-hash-generation``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+  Set this property to ``false`` when running Presto C++ workers 
+  because Velox does not support optimized hash generation, 
+  instead performing this optimization adaptively.
+
+``regex-library``
+^^^^^^^^^^^^^^^^^
+
+* **Type:** ``type``
+* **Allowed values:** ``RE2J``
+* **Default value:** ``JONI``
+
+  Only `RE2J <https://github.com/google/re2j>`_ is currently supported by Velox.
+
+``use-alternative-function-signatures``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+  Some aggregation functions use generic intermediate types which are 
+  not compatible with Velox aggregation function intermediate types. One  
+  example function is ``approx_distinct``, whose intermediate type is 
+  ``VARBINARY``. 
+  Stubs in the Presto codebase provide alternative stub implementations 
```

Comment:
I would prefer to no use the word stub as it has other uses such as a placeholders.
Let's say `Presto codebase provides alternate function signatures optimized for Velox.`

Reply:
I don't think `optimized for Velox` is accurate.  This is required to work, so I think `compatible with Velox` is more accurate.

`This property provides function signatures for built-in aggregation functions which are compatible with Velox.`

Reply:
Makes sense!